pkg/server/statements.go

```diff
@@ -89,13 +91,21 @@ func (s *statusServer) StatementsLocal(ctx context.Context) (*serverpb.Statement
 	}
 
 	for i, stmt := range stmtStats {
+		stmts, err := parser.Parse(stmt.Key.Query)
+		if err != nil {
+			return nil, err
+		}
+		pretty := tree.PrettyWithOpts(stmts[0], 80, false, 4)
+		prettyNarrow := tree.PrettyWithOpts(stmts[0], 40, false, 2)
```

Comment:
As I write in my comment this call should not be made here and instead exist in a separate RPC endpoint.

Also you are using an obsolete version. You should use the new version (with a rebase) and pass the argument `simplify = true` to get rid of the extraneous parentheses.